### ISSUE TITLE
error when using economy api 

### ISSUE THREAD

**Issue by Lavitratyagi** [onebone/EconomyS#461]
'EconomyAPI': Incompatible API version (plugin requires one of: 3.0.0-ALPHA7, 3.0.0-ALPHA8, 3.0.0-ALPHA9) this error is coming when i am using it in pmmp latest version

**Comment by Brougud** [comment]
yeah and then when you try to load in game it will keep you on the 'connecting to external server' page?
